modules/affile/stdin.c

```diff
@@ -58,7 +58,7 @@ file_opener_for_stdin_new(void)
 LogDriver *
 stdin_sd_new(GlobalConfig *cfg)
 {
-  AFFileSourceDriver *self = affile_sd_new_instance("-", cfg);
+  AFFileSourceDriver *self = affile_sd_new_instance("/dev/stdin", cfg);
```

Comment:
What problem does this patch solve?

I'm asking because `/dev/stdin` is not POSIX, so I don't know if this would work outside Linux.
It's basically a symlink to `/proc/self/fd/0`.

Reply:
https://github.com/axoflow/axosyslog/blob/bff69a5dfbb064e354c0671fe9b55a3673335afe/modules/affile/file-opener.c#L175

The default `-` character for path in this case is easy to miss when debugging trace logs, something closer to `stdin` communicates this intent better I think.

Reply:
Sorry, just seeing the second comment.
Of course, will do it in a minute, thanks for double checking it thoroughly! :rocket: